casr/src/bin/casr-python.rs

```diff
@@ -186,7 +175,7 @@ fn main() -> Result<()> {
         }
     } else {
         // Call sub tool
```

Comment:
Update comments here and in similar places

Reply:
done